src/content/resources/kotlin/jvm/kutils-guide.md

```diff
@@ -0,0 +1,146 @@
+
+If, however, you're kind of bored, and wish I'd get on with the damned tutorial---
+
+Then I've got news for you.
+
+Because that's what we're doing.
+
+### Goodbye, println, hello KUtils!
+
+So, let's swap out that measly little `Hello, world!` for a KUtils login, shall we?
+
+So, gut the contents of `main`, and replace them with this:
+
+```kotlin
+startBot("your-bot-token-goes-here") {
+  configure {
+    prefix = "!"
+    reactToCommands = true
+    deleteMode = PrefixDeleteMode.None
+  }
+}
+```
+
+Now, don't worry, if IntelliJ complains that it can't find any of the things there, click on where it goes red, and hit <kbd>alt+enter</kbd> until there are none left. Now let me break this down.
+
+- `startBot` calls KUtils' start function, using the string we're passing it there (which you need to replace with a Discord bot token. If you don't know what that is or how to get it, use some Google-fu.)
+- `configure` is a KConfiguration safe type constructor, which we use to configure the bot's features.
+- `reactToCommands` tells the bot to give us a little set of emoji eyes whenever we use a command to tell us it's recognised it.
+- `deleteMode` isn't important, and the reasoning for it being there isn't either.
+
+So, is that it? Well, yeah. It is. If you hit run again, you should be greeted with a set of warnings and messages in the output window again. However, if you invite the bot to a server, you'll see that it's now online! Awesome!
+
+Go into that server and type `!help`, and you should be greeted with a cute little embed telling you how the help command works!
+
+## Commands
+
+### Oi! Don't be so bossy with those command words!
+
+So, if you've followed the guides thus far, you might be shouting, "Oi! Hayden! Get off your lazy bum and tell me how to make my own commands!"
+
+Alright, alright, simmer down. We'll do that now.
+
+So, to get started, find the folder your bot's main `.kt` file is in, and right click, go to 'New', then 'Package'. Type 'commands' in the box and smash that enter key like your finger is a missile. (although... not if you're on a laptop... that... that could end badly.)
+
+Make a new Kotlin Script in that folder called `UtilityCommands`. We're gonna first off declare that we're making a set of commands.
+
+```kotlin
+@CommandSet("Utility")
+```
+
+This creates a new CommandSet called "Utility". Although you may notice it's got a red line under it. That's just IntelliJ screaming at you because you don't have a function containing all the sweet, sweet commands it has to run. So let's give it one.
+
+```kotlin
+@CommandSet("Utility")
+fun utilityCommands() = commands {
+
+}
+```
+
+Note how this has a weird ending. It doesn't go straight into a function, this goes into a `commands` block. Huh. Well, believe me, it makes it very simple. Because now all you need to do is add this:
+
+```kotlin
+command("Ping") {
+    description = "Pong!"
+    execute {
+        it.respond("Pong!")
+    }
+}
```

Comment:
I think it might be a good idea to show that the `command("Ping")` portion bit goes in the `utilityCommands` function, feels like it could be confusing but other than it looks good

Reply:
Done. See my latest commit.